src/ca65/main.c

```diff
@@ -307,6 +307,14 @@ static void SetSys (const char* Sys)
             NewSymbol ("__OSIC1P__", 1);
             break;
 
+        case TGT_CREATIX_EMUL:
+            NewSymbol ("__CREATIX_EMUL__", 1);
+            break;
+            
+        case TGT_CREATIX_NATIVE:
```

Comment:
Why do you need emul and native libs? You want to have a runtime lib for "creatix", correct? If it uses emul features (and I guess it does, since that was your reason to add it), build it with "emul".